bundles/edu.kit.kastel.sdq.eclipse.common.core/src/edu/kit/kastel/eclipse/common/core/AssessmentController.java

```diff
@@ -58,6 +58,21 @@ protected AssessmentController(GradingSystemwideController systemWideController,
 		this.annotationDAO = new AnnotationDAO();
 		this.gradingDAO = loadGradingDAO();
 
+		try {
+			ExerciseConfig exerciseConfig = this.gradingDAO.getExerciseConfig();
+			if (!exerciseConfig.getAllowedExercises().isEmpty() && !exerciseConfig.getAllowedExercises().contains(this.exercise.getExerciseId())) {
+				// using interaction handler of the system wide controller, as the own
+				// interaction handler is not set during the constructor
+				systemWideController.getViewInteractionHandler().warn("""
+						You are using a configuration file not designed for this exercise!\n
+						Your file is \"%s\", however you are correcting exercise \"%s\"!\n
+						Please double check your settings!
+						""".formatted(exerciseConfig.getShortName(), this.exercise.getShortName()));
+			}
+		} catch (IOException e) {
+			this.error("Exercise Config not parseable: " + e.getMessage(), e);
```

Comment:
> mmh it could be the case that something has crashed due to the the missing ",".

We could use the handler from the systemwideController to inform about the broken config when an assessment is started.

Reply:
I think this should be done in another PR. As those parse-issues presumably could be handled once, which might invole a bit more logic.